core/src/assets/defaultConfig.json

```diff
@@ -61,7 +61,10 @@
         }
       },
       "PROMETHEUS": {
-        "isEnabled": true
+        "isEnabled": true,
+        "config": {
+          "path": "api/v1/namespaces/kyma-system/services/monitoring-prometheus:web/proxy/api/v1"
```

Comment:
do we need actually the whole path?
both the prefix (`api/v1/namespaces/kyma-system/services/`) and the suffix (`/proxy/api/v1`) are very unlikely to change - as they're part of the kubernetes and prometheus implementations respectively.
The only two parts that can easily change on a per-cluster basis are service name and port name. I feel that would be enough. Either as a single value (`monitoring-prometheus:web`) or two, one for service name (`monitoring-prometheus`) one for port (`web`)